rnd/autogpt_builder/sentry.edge.config.ts

```diff
@@ -8,6 +8,8 @@ import * as Sentry from "@sentry/nextjs";
 Sentry.init({
   dsn: "https://fe4e4aa4a283391808a5da396da20159@o4505260022104064.ingest.us.sentry.io/4507946746380288",
 
```

Comment:
I’d like to suggest doing the before send instead of the whole thing and see how you feel about that

Reply:
I don't understand what you are getting at?